src/pages/AgentNew.tsx

```diff
@@ -0,0 +1,51 @@
+import { useNavigate, Link } from 'react-router-dom'
+import { Button, FloatingLabel, Form } from 'react-bootstrap'
+import icon from '../images/icon.svg'
+
+export default function AgentNew() {
+  const navigate = useNavigate()
+
+  return (
+    <div>
+      <a role="button" onClick={() => navigate(-1)}>
+        <i className="bi bi-chevron-left align-middle" />
+        Back
+      </a>
+      {/* place text horizontal to 'Back' nav link? */}
+      <h2 className="mb-4 fw-normal text-center">Add listing</h2>
+      {/* switch to officer icon */}
+      <div className="mb-4 text-center">
+        <img src={icon} alt="ProPro logo" style={{ width: 50 }} />
+      </div>
```

Comment:
Here's how I might go about making the icon circle
```suggestion
      <div className="d-flex justify-content-center mb-3">
        <div
          className="d-flex justify-content-center align-items-center bg-white rounded-circle"
          style={{ width: 80, height: 80 }}
        >
          <img src={icon} alt="ProPro logo" width="50%" />
        </div>
      </div>```

Reply:
So helpful! (I had added a to-do comment to figure out the white circle background.) Comparing the size ratios on the app version, it seems the icon should be a bit bigger here, but that's a detail that can be modified later if needed.